addons/hr_recruitment/tests/test_recruitment.py

```diff
@@ -6,6 +6,22 @@
 @tagged('recruitment')
 class TestRecruitment(TransactionCase):
 
+    def test_infer_aplicant_lang_from_context(self):
+        # Prerequisites
+        self.env['res.lang']._activate_lang('pl_PL')
+        self.assertGreaterEqual(len(self.env['res.lang'].get_installed()), 2, "Need more langagues than 'pl_PL' installed")
```

Comment:
I think it's best that you ensure that more than 2 languages are installed rather than having a check for it, and same for the default as well :)

Reply:
True, thanks for having a look ;)